reports-scheduler/src/main/java/com/amazon/opendistroforelasticsearch/reportsscheduler/rest/ReportsSchedulerRestHandler.java

```diff
@@ -0,0 +1,161 @@
+        if (request.method().equals(RestRequest.Method.POST)) {
+            // compose JobParameter object from request
+            String id = request.param("id");
+
+            //TODO: add start_time and Time_unit from request params or request body
+
+            //TODO: parse the content not params.
+            /**
+             * {
+             *
+             *     "name": "Flights monthly report",
+             *     "enabled": true, //boolean, default=true, required by ScheduledJobParameter
+             *     "schedule": {
+             *         "interval": {
+             *             "period": 1,
+             *             "unit": "MINUTES",
+             *             "start_time": 1553112384
+             *         }
+             *
+             *       },
+             *     "report_definition_id":"23dasdfdasdas"
+             *  }
+             */
+
+            if(id == null) {
+                throw new IllegalArgumentException("Must specify id");
+            }
+
+            IndexRequest indexRequest = new IndexRequest()
+                    .index(ReportsSchedulerPlugin.JOB_INDEX_NAME)
+                    .id(id)
+                    .source(request.requiredContent(), XContentType.JSON);
+
+            return restChannel -> {
+                // index the job parameter
+                client.index(indexRequest, new ActionListener<IndexResponse>() {
+                    @Override
+                    public void onResponse(IndexResponse indexResponse) {
+                        try {
+                            RestResponse restResponse = new BytesRestResponse(RestStatus.OK,
+                                    indexResponse.toXContent(JsonXContent.contentBuilder(), null));
+                            restChannel.sendResponse(restResponse);
+                        } catch(IOException e) {
+                            restChannel.sendResponse(new BytesRestResponse(RestStatus.INTERNAL_SERVER_ERROR, e.getMessage()));
+                        }
+                    }
+
+                    @Override
+                    public void onFailure(Exception e) {
+                        restChannel.sendResponse(new BytesRestResponse(RestStatus.INTERNAL_SERVER_ERROR, e.getMessage()));
+                    }
+                });
+            };
+        } else if (request.method().equals(RestRequest.Method.DELETE)) {
+            // delete job parameter doc from index
+            String id = request.param("id");
+            DeleteRequest deleteRequest = new DeleteRequest()
+                    .index(ReportsSchedulerPlugin.JOB_INDEX_NAME)
+                    .id(id);
+
+            return restChannel -> {
+                client.delete(deleteRequest, new ActionListener<DeleteResponse>() {
+                    @Override
+                    public void onResponse(DeleteResponse deleteResponse) {
+                        restChannel.sendResponse(new BytesRestResponse(RestStatus.OK, "Job deleted."));
```

Comment:
what does "Job deleted." means to caller?

Reply:
In the latest change, it's returning something like ""Job deleted. Job id: <jobID>". Does that make sense? 
reports-scheduler/src/main/java/com/amazon/opendistroforelasticsearch/reportsscheduler/rest/handler/ReportsScheduleActionHandler.java